packages/core/@generated/graphql/index.ts

```diff
@@ -1092,15 +1092,15 @@ export type ProductDetailsFragment_ProductFragment = {
   }>
 }
 
-export type ProductGalleryQueryQueryVariables = Exact<{
+export type BrowserProductGalleryQueryQueryVariables = Exact<{
```

Comment:
```suggestion
export type ClientProductGalleryQueryQueryVariables = Exact<{
```

I prefer using `client` as prefix 🙈

Reply:
Nice! changes made [here](https://github.com/vtex/faststore/pull/1938/commits/516a436038954f637104155931af5d1a37659a2d).